doc/mgmt/SONiC_PIM_SM_HLD.md

```diff
@@ -0,0 +1,1036 @@
++       |     |  |  |     +--ro spt-action?                 enumeration
++       |     |  |  |     +--ro spt-infinity-prefix-list?   -> /oc-acl:acl/acl-sets/acl-s
+```
+module: sonic-pim-show
+```diff
+  rpcs:
+    +---x show-pim
+       +---w input
+       |  +---w vrf-name?                      string
+       |  +---w address-family?                enumeration
+       |  +---w query-type                     show-pim-query-type
+       |  +---w (type)?
+       |     +--:(RPF)
+       |     |  +---w rpf?                     boolean
++      |     +--:(RPINFO)
++      |     |  +---w rpinfo?                  boolean
++      |     |  +---w rpinfo-per-group-addr?   inet:ip-address
++      |     +--:(ASSERT)
++      |        +---w assert?                  boolean
+       +--ro output
+          +--ro status?   string
+module: sonic-pim
+  +--rw sonic-pim
+     +--rw PIM_GLOBALS
+     |  +--rw PIM_GLOBALS_LIST* [vrf-name address-family]
+     |     +--rw vrf-name                    union
+     |     +--rw address-family              enumeration
+     |     +--rw join-prune-interval?        uint16
+     |     +--rw keep-alive-timer?           uint16
+     |     +--rw ssm-ranges?                 -> /srpolsets:sonic-routing-policy-sets/PREFIX_SET/PREFIX_SET_LIST/name
+     |     +--rw ecmp-enabled?               boolean
+     |     +--rw ecmp-rebalance-enabled?     boolean
++    |     +--rw spt-action?                 spt-switchover-action-type
++    |     +--rw spt-infinity-prefix-list?   -> /srpolsets:sonic-routing-policy-sets/PREFIX_SET/PREFIX_SET_LIST/name
++    +--rw PIM_GLOBALS_RENDEZVOUS_POINT
++    |  +--rw PIM_GLOBALS_RENDEZVOUS_POINT_LIST* [vrf-name address-family rp-address]
++    |     +--rw vrf-name            union
++    |     +--rw address-family      enumeration
++    |     +--rw rp-address          inet:ip-address
++    |     +--rw prefix-list-name?   -> /srpolsets:sonic-routing-policy-sets/PREFIX_SET/PREFIX_SET_LIST/name
+     +--rw PIM_INTERFACE
+        +--rw PIM_INTERFACE_LIST* [vrf-name address-family interface]
+           +--rw vrf-name          union
+           +--rw address-family    enumeration
+           +--rw interface         union
+           +--rw mode?             enumeration
+           +--rw dr-priority?      uint32
+           +--rw hello-interval?   uint8
+           +--rw bfd-enabled?      boolean
+           +--rw bfd-profile?      string
+```
+### 3.5.2 CLI
+
+#### 3.5.2.1 Global Configuration Commands
+
+##### 3.5.2.1.1 ip pim rp-address 
+
+
+| ip pim rp-address  |        Configure a static PIM rendezous point (RP) address for a group.                                                                     |
+| ------------------------------------------------ | ------------------------------------------------------------ |
+| **Syntax**                                                                    | ip pim [vrf vrf-name] rp-address < address > [{ prefix-list prefix-list-name}]                      |
```

Comment:
Seems FRR has support for providing either prefix list or the group address range. Do we also allow the same in sonic-cli?

sonic(config)# ip pim rp 10.1.1.1
<cr>
A.B.C.D/M Group Address range to cover
prefix-list group prefix-list filter

Reply:
No, as per ISCLI standard we are retaining only the prefix-list and not the groups in RP command.